src/main/java/gregtech/api/recipes/RecipeMaps.java

```diff
@@ -197,7 +197,8 @@ public final class RecipeMaps {
     @ZenProperty
     public static final RecipeMap<AssemblyLineRecipeBuilder> ASSEMBLY_LINE_RECIPES = new RecipeMapAssemblyLine<>(
             "assembly_line", new AssemblyLineRecipeBuilder(), AssemblyLineUI::new)
-                    .onRecipeBuild(AssemblyLineManager::createDefaultResearchRecipe);
+                    .onRecipeBuild(gregtechId("default_research_recipe"),
```

Comment:
Why does this one use `onRecipeBuild` instead of `onBuild`?

Reply:
This RecipeMap does not utilize `RecipeMapBuilder`, and therefore must call `RecipeMap#onRecipeBuild` instead.